src/sys/unix.rs

```diff
@@ -560,6 +621,35 @@ impl Socket {
         Ok(n as usize)
     }
 
+    #[cfg(not(target_os = "redox"))]
+    pub fn send_vectored(&self, bufs: &[IoSlice<'_>], flags: c_int) -> io::Result<usize> {
+        let mut msg: libc::msghdr = unsafe { std::mem::zeroed() };
```

Comment:
Nit: same as above for the `libc::msghdr`.

Reply:
Done :)